example/grid_search_example/gs_config.yml

```diff
@@ -5,6 +5,11 @@ global_config:
   model_prediction_key: &model_prediction_key_anchor model_prediction_key
   postprocessing_argmax_key: &postprocessing_argmax_key_anchor postprocessing_argmax_key
 
+model_info:  
+  model_description: "Testing 1 2 3 Hopefully it works."
+  model_version: 1.0.0
+  source_repo: "Github repo link Ex: https://github.com/mlgym/mlgym/tree/dev"
```

Comment:
would be nice if we could retreive the source repo automatically in the future

Reply:
Yes, I will try to give it another go and see if I ca get it dynamically from a python package if not I will keep it as it is and create an issue for further improvement.